model/accounts_mgmt/v1/root_resource.model

```diff
@@ -148,4 +148,9 @@ resource Root {
 	locator QuotaAuthorizations {
 		target QuotaAuthorizations
 	}
+
+	// Reference to the resource that manages billing models.
+    locator BillingModels {
+    	target BillingModels
+    }
```

Comment:
not critical probably, but you're mixing spaces and tabs which makes the indentation here a bit weird
same goes for other parts in this PR

Reply:
Ahh, seems like my IDE has some issues. Replaced space with tabs.